### ISSUE TITLE
Multiple config files

### ISSUE THREAD

**Issue by MatthieuCoder** [ory/hydra#2767]
**Is your feature request related to a problem? Please describe.**

I'm trying to run Hydra in Kubernetes using Vault as a DSN provider; for other containers i just change the startup script and load the environment variables using `source`, but the hydra container does not includes sh

**Describe the solution you'd like**

I'm thinking about multiple config files (like docker-compose does), so I can load another file from vault containing the secret

**Describe alternatives you've considered**

Loading secrets from files could be another solution, but this seems over complicated compared to the first solution

**Comment by zepatrik** [comment]
Can't you start the container with the env vars already set? That is the intended way of doing this.

**Comment by MatthieuCoder** [comment]
No, The vault-injector only creates shared volumes using a init-container (and a sidecar for updating the file)

**Comment by zepatrik** [comment]
I see, maybe @Demonsthere or @piotrmsc know some workaround? Otherwise, I think there is no problem with supporting multiple config files, it is just a bit work to properly implement it.

**Comment by zepatrik** [comment]
Just looked into the code, it should already support multiple config files :thinking: 
Provide them with multiple `hydra serve -c foo.yml -c bar.yml`

**Comment by MatthieuCoder** [comment]
That's sweet! Should this be an option in the helm chart ?

**Comment by zepatrik** [comment]
Probably, if it is not yet possible.
